### PR TITLE
Move Enterprise metrics content

### PR DESCRIPTION
**Pull request by ZPain8464**
This PR:
- Moves the existing "Metrics" content to a new page: `/docs/guides/configure-metrics`.
- Adds updated screenshots.
- Updates content.

We won't merge this PR until the new Zero Metrics content is ready in another PR.

Related to https://github.com/pomerium/documentation/issues/1618.

### CODE REVIEW

File: content/docs/guides/configure-metrics.mdx [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+   If you're running Pomerium Enterprise in a distributed environment where the IP address is unknown at the time of deployment, you can use the resolvable fully qualified domain name (FQDN) of the Pomerium host (for example, `pomerium0.internal.example.com`). Or, you can override this key with the [`METRICS_ADDRESS`](/docs/reference/metrics#metrics-address) environment variable.
+
+   We do not recommend exposing this endpoint to public traffic as it can contain potentially sensitive information.
+
+   :::
+
+1. In your Pomerium Enterprise configuration file, define the `metrics_addr` key to a network interface or port. For example:
+
+   ```yaml title="pomerium-enterprise.yaml"
+   metrics_addr: 127.0.0.1:9092
```
Comment (ZPain8464) [inline content/docs/guides/configure-metrics.mdx:51]: @wasaga I can't find this setting anywhere, only the `metrics_address` setting. Is this setting still supported? Do we need to add it to the Enterprise configuration page?

File: content/docs/guides/configure-metrics.mdx [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+### External Prometheus
+
+1. In your Prometheus configuration file, add [instances](https://prometheus.io/docs/concepts/jobs_instances/#jobs-and-instances) for Pomerium and Pomerium Enterprise:
+
+   ```yaml title="prometheus.yaml"
+   - job_name: 'Pomerium'
+      scrape_interval: 30s
+      scrape_timeout: 5s
+      static_configs:
+         - targets: ['127.0.0.1:9901']
```
Comment (ZPain8464) [inline content/docs/guides/configure-metrics.mdx:68]:
```suggestion
         - targets: ['127.0.0.1:9091']
```

File: content/docs/guides/configure-metrics.mdx [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+   - job_name: 'Pomerium Enterprise'
+      scrape_interval: 30s
+      scrape_timeout: 5s
+      static_configs:
+         - targets: ['127.0.0.1:9902']
```
Comment (ZPain8464) [inline content/docs/guides/configure-metrics.mdx:73]:
```suggestion
         - targets: ['127.0.0.1:9092']
```

File: content/docs/guides/configure-metrics.mdx [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+
+   ```
+
+1. [Reload](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#configuration) the Prometheus configuration:
+
+   ```bash
+   curl -i -XPOST path.to.prometheus:port/-/reload
+   ```
+
+1. In your Pomerium Enterprise configuration file, define the [`prometheus_url`](/docs/enterprise/configure#prometheus-url) key to point to your Prometheus instance:
```
Comment (ZPain8464) [inline content/docs/guides/configure-metrics.mdx:83]:
```suggestion
1. In your Pomerium Enterprise configuration file, define the [`prometheus_url`](/docs/enterprise/configure#prometheus-url) key to point to your Prometheus instance. The example below uses the default [Prometheus port](https://prometheus.io/docs/introduction/first_steps/).
```

File: content/docs/guides/configure-metrics.mdx (hunk 1 above)
Comment (wasaga) [reply content/docs/guides/configure-metrics.mdx:51]: `metrics_addr` is correct. there's no `metrics_address` setting in the console 

```go
flags.StringVar(&serveOptions.metricsAddr, "metrics-addr", "127.0.0.1:9092", "address to expose console metrics in Prometheus format")
```

**Review by wasaga — APPROVED** [review]

**Pull request merged**